addons/payment/views/portal_templates.xml

```diff
@@ -187,11 +187,13 @@
                                 <t t-set="value" t-value="tx.reference"/>
                                 <t t-set="include_separator" t-value="True"/>
                             </t>
-                            <t t-call="payment.summary_item">
-                                <t t-set="name" t-value="'method'"/>
-                                <t t-set="label">Payment Method</t>
-                                <t t-set="value" t-value="tx.payment_method_id.name"/>
-                                <t t-set="include_separator" t-value="True"/>
+                            <t t-if="tx.payment_method_code!='unknown'">
```

Comment:
In `pos_online_payment` too; maybe others? Check for modules that call `payment.summary_item`.